assets/src/edit-story/components/panels/design/preset/utils.js

```diff
@@ -204,3 +210,30 @@ export function areAllType(elType, selectedElements) {
     selectedElements.every(({ type }) => elType === type)
   );
 }
+
+export function getOpaqueColor(preset) {
+  const { color } = preset;
+  return {
+    color: {
+      ...color,
+      a: 1,
+    },
+  };
+}
+
+export function getPanelInitialHeight(isColor, presets) {
+  const rowHeight = isColor ? SAVED_COLOR_SIZE : SAVED_STYLE_HEIGHT;
+  // Includes the helper text and button for saving a color.
+  const emptyColorsHeight = 140;
```

Comment:
These seem like good candidates for a couple of small unit tests

Reply:
Good idea. This will change more with Global + Local colors, was thinking of adding more test coverage then but doesn't harm adding it already now for the utils.